Amethyst/Managers/LayoutManager.swift

```diff
@@ -40,7 +40,39 @@ enum LayoutManager {
         }
     }
 
-    static func availableLayoutStrings() -> [String] {
+    static func layoutNameForKey(_ layoutKey: String) -> String? {
```

Comment:
This function is perhaps a little ugly, but patterned directly after the `layoutForKey()` function above it (which is arguably even more ugly). It might be even better to refactor both functions to create the mappings automatically, without explicitly retyping all of the layout keys (if that's possible). Perhaps the `layoutClasses` list currently created in the `availableLayoutStrings()` function could be raised to a full static member, and then these `layoutForKey()` and `layoutNameForKey()` functions could be somehow expressed in terms of maps over that list.

Reply:
I think making the layout list static and mapping over it is a good idea.

Reply:
I just pushed a commit doing that refactor, and indeed the code is much more elegant.